lib/asset/transferasset.js

```diff
@@ -215,6 +215,61 @@ class TransferAsset {
         return data;
     }
 
+    /**
+     * Retrieves the timestamp of when the asset started transferring. Will be 0 if no
+     * start time is available.
+     *
+     * @returns {number} Timestamp.
+     */
+    get transferStartTime() {
+        const { transferStart = 0 } = this[PRIVATE];
+        return transferStart;
+    }
+
+    /**
+     * Sets the timestamp of when the asset started transferring.
+     *
+     * @param {number} startTime Timestamp.
+     */
+    set transferStartTime(startTime) {
+        this[PRIVATE].transferStart = startTime;
+    }
+
+    /**
+     * Retrieves the timestamp of when the asset finished transferring. Will be 0 if no
+     * end time is available.
+     *
+     * @returns {number} Timestamp.
+     */
+    get transferEndTime() {
```

Comment:
Should `transferEndTime` be added to the constructor?

Reply:
I suppose if there's never a check for 0 or null specifically (we only check falsy), we don’t need to add it to the constructor. I can't really think of a use case without looking through the repo a bit more. No concerns on my end, thanks for this work!!

Reply:
> we only check falsy

`0`, `null`, and `undefined` will all be treated as `false` in a falsy check, so both of those would end up as `0` for the return value.